src/no/runsafe/nchannels/database/ChannelMembershipRepository.java

```diff
@@ -44,17 +50,23 @@ public ISchemaUpdate getSchemaUpdateQueries()
 		return update;
 	}
 
+	public Boolean isEmptyChannel(String channel)
+	{
+		List<String> stringIds = database.queryStrings("SELECT player FROM nchannel_members WHERE channel=?", channel);
+		return stringIds.isEmpty();
+	}
+
 	public List<String> getPlayerChannels(IPlayer player)
 	{
 		return database.queryStrings("SELECT channel FROM nchannel_members WHERE player=?", player.getUniqueId().toString());
 	}
 
-	public List<UUID> getChannelPlayers(String channel)
+	public List<IPlayer> getChannelPlayers(String channel)
 	{
 		List<String> stringIds = database.queryStrings("SELECT player FROM nchannel_members WHERE channel=?", channel);
-		List<UUID> playerIds = new ArrayList<UUID>();
+		List<IPlayer> playerIds = new ArrayList<>();
 		for(String playerStringId : stringIds)
```

Comment:
Add a method to IPlayerProvider that converts a List of UUID into a List of IPlayer